src/pages/data-management/index.jsx

```diff
@@ -25,34 +27,46 @@ const DataManagementPage = ({ experimentId, experimentData, route }) => {
 
   const [projectsList, setProjectsList] = useState([]);
   const [newProjectModalVisible, setNewProjectModalVisible] = useState(true);
+  const [activeProjectIdx, setActiveProjectIdx] = useState(0);
 
   useEffect(() => {
     if (projectsList.length) {
       setNewProjectModalVisible(false);
     }
   }, [projectsList]);
 
-  const createNewProject = () => {
+  const createNewProject = (newProjectName) => {
+    const newProject = {
+      name: newProjectName,
+      createdDate: moment().local().format('DD MMM YYYY, HH:mm:ss [GMT]Z'),
+      lastModified: moment().local().format('DD MMM YYYY, HH:mm:ss [GMT]Z'),
+      numSamples: 0,
+      lastAnalyzed: '-',
+    };
+
+    setProjectsList([...projectsList, newProject]);
+    setActiveProjectIdx(projectsList.length);
     setNewProjectModalVisible(false);
   };
 
   const TILE_MAP = {
     'Projects List': {
-      toolbarControls: [<RemoveButton />],
+      toolbarControls: [],
       component: (width, height) => (
-        <div width={width} height={height}>
+        <Space direction='vertical' style={{ width: '100%', overflowY: 'scroll' }}>
           <Button type='primary' block onClick={() => setNewProjectModalVisible(true)}>
             Create New Project
           </Button>
-        </div>
+          <ProjectsList projects={projectsList} height={height} activeProjectIdx={activeProjectIdx} />
+        </Space>
       ),
     },
     'Data Management': {
-      toolbarControls: [
-        <RemoveButton />,
-      ],
```

Comment:
This also remove the ability for users to close the project management window

Reply:
Why do we want to remove this ability?